app/js/app.js

```diff
@@ -73,11 +75,14 @@ gaia-modal {
   }
 
   function boot(manifestURL) {
-    var mainController = new MainController({
-      manifestURL: manifestURL
-    });
+    if (manifestURL === SYSTEM_APP) {
+      return new GestureController();
```

Comment:
I think that this name could be improved, as it doesn't tell me much about what's really happening. Maybe something like ```SystemTouchEventController```.